rewrite-gradle/src/main/java/org/openrewrite/gradle/UpgradeDependencyVersion.java

```diff
@@ -265,7 +264,7 @@ public J visitMethodInvocation(J.MethodInvocation method, ExecutionContext ctx)
                         String declaredVersion = null;
 
                         for (Expression e : m.getArguments()) {
-                            if (!(e instanceof G.MapEntry)) {
+                            if (!(e instanceof J.Assignment)) {
                                 continue;
                             }
                             J.Assignment assignment = (J.Assignment) e;
```

Comment:
This seemed the wrong `instanceof` type; when changed accordingly we now pass through the for loop.